assets/js/components/ExecutionResults/CheckResultOutline.jsx

```diff
@@ -36,17 +44,49 @@ function CheckResultOutline({
         isAgentCheckError || metExpectations < expectExpectationsCount,
     };
   });
+};
+
+const expectSameExpectations = (expectationList) =>
+  expectationList.filter(isExpectSame);
+
```

Comment:
Inline this please

Reply:
Following the other comment and renaming to `getExpectSameStatements`